include/voltorb_flip/voltorb_flip_work.h

```diff
@@ -37,32 +37,31 @@ typedef struct VoltorbFlipAppWork {
     PlayerProfile *profile;
     NARC *narc;
     BgConfig *bgConfig;
-    void *unk1C;
+    void *listMenuSpawner;
     TouchscreenListMenu *menu;
     Window windows[17];
     LISTMENUITEM *menuItems[2];
-    void *unk13C;
+    YesNoPrompt *yesNoPrompt;
     String *string;
     SpriteSystem *spriteRenderer;
     SpriteManager *spriteGfxHandler;
-    ManagedSprite *unk14C[0xd];
+    ManagedSprite *sprites[0xd];
     MsgData *msgData;
     MessageFormat *msgFmt;
     u16 unk188[10][8];
-    u8 unk228;
+    u8 menuInputState;
     u8 unk229;
     u8 unk22A;
     u8 printerId;
     u16 payoutDisplay;
-    u8 unk22E[0x2];
-    int unk230;
-    int unk234;
-    int unk238;
+    BOOL hasPlayedOneLeve;
+    BOOL printedCanGetMoreCoinsMessage;
+    BOOL memoOpen;
     struct WorkflowEngine *workflow;
-    struct Ov122_021E8CFC *unk240;
-    GameState *game;
-    Ov122_021E70B8 unk248;
-    Ov122_021E7888 unk25C;
+    VoltorbFlipInputHandler *inputHandler;
+    VoltorbFlipGameState *game;
+    VFCardFlipAnim cardFlipControl;
+    u8 screenIsDimmed;
 } VoltorbFlipAppWork;
```

Comment:
Vestigial: We want to phase out 'work' terminology since it doesn't mean anything. `VoltorbFlipApp` is fine on its own.